content/community/index.fr.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Node.js Community
```

Comment:
Note that you still need to update the `title` field back to the original one and add the `displayTitle` one.